NRExoPlayerTracker/src/main/java/com/newrelic/videoagent/exoplayer/tracker/NRTrackerExoPlayer.java

```diff
@@ -91,28 +93,28 @@ public Map<String, Object> getAttributes(String action, Map<String, Object> attr
     /**
      * Get player name.
      *
-     * @return Attribute.
+     * @return Player name from Media3 library.
      */
     public String getPlayerName() {
-        return "ExoPlayer2";
+        return MediaLibraryInfo.TAG;
     }
 
     /**
      * Get player version.
      *
-     * @return Attribute.
+     * @return Player version from Media3 library.
      */
     public String getPlayerVersion() {
-        return "2.x";
+        return MediaLibraryInfo.VERSION;
     }
 
     /**
      * Get tracker name.
      *
-     * @return Atribute.
+     * @return The simple class name of this tracker.
      */
     public String getTrackerName() {
-        return "ExoPlayer2Tracker";
+        return this.getClass().getSimpleName();
```

Comment:
I see the point of using the class name as id here, but many people is currently using this tracker and generating data and building dashboards based on this data. Changing this id now could cause a disruption and potentially break dashboards that rely on filters using this id at is it now.

Reply:
```suggestion
        return "ExoPlayer2Tracker";
```
I makes sense, let me change it back here. :)